src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -104,12 +113,15 @@ public boolean hasClashingLesson(Lesson lesson, Lesson lessonToIgnore) {
 
     /**
      * Returns true if any of the specified lessons clashes with existing lesson in the address book.
+     *
+     * @param lessons The lessons we want to check.
+     * @return True if any of the given lessons clash with any existing lessons.
      */
     public boolean hasClashingLesson(Iterable<Lesson> lessons) {
         requireAllNonNull(lessons);
         return entries.hasClashes(lessons);
     }
-
+    //@@author
```

Comment:
Can help me add? Then add empty @@author before addPerson

```suggestion
    //@@author eeliana
```

Reply:
If you not claiming then no need

Reply:
decided not to claim